olp-cpp-sdk-core/include/olp/core/client/OlpClient.h

```diff
@@ -95,17 +104,18 @@ class CORE_API OlpClient {
   /**
    * @brief Executes the REST request through the network stack in a blocking
```

Comment:
```suggestion
   * @brief Executes the HTTP request through the network stack in a blocking
```
Not related to the change but it'd be nice to fix

Reply:
Changed